core/tests/loadnext/src/sdk/operations/withdraw.rs

```diff
@@ -70,7 +70,7 @@ where
                     .get_bridge_contracts()
                     .await
                     .map_err(|err| ClientError::NetworkError(err.to_string()))?;
-                default_bridges.l2_erc20_default_bridge
+                default_bridges.l2_erc20_default_bridge.unwrap()
```

Comment:
is it safe?

Reply:
It is not safe for custom chains, it is safe for Era though